scripts/Install-VSCode.ps1

```diff
@@ -92,27 +105,38 @@
 #>
 [CmdletBinding()]
 param(
+    [parameter()]
+    [ValidateSet("stable","insider")]
+    [string]$BuildEdition = "stable",
```

Comment:
Please separate each parameter with a blank line - ditto between lines 113 and 114 - for consistency (see Start-EditorServices.ps1).

Reply:
Changes made as per comment